### ISSUE TITLE
This is 100% A Virus (Certified by Virustotal)

### ISSUE THREAD

**Issue by DAVIDEGAMERS33** [Kalejin/DCSB#143]
Hello everyone! I saw this soundboard that contains 5 Malicious flag

Bkav Pro is a Vietnamese "Antivirus", I think the antivirus is a FAKE ONE, and it's not the official one.

As Cyberreason does not connect with this Antivirus

Try to search on Google images "Bkav Pro + Cyberreason" https://prnt.sc/m0xuzAZBuXSF

The same thing with "Securage + VBA32", I think another "Antivirus"
As it contains a Trojan in it

Then a flag of the webroot, weird of another "Antivirus"

I'm not a true professional guy on this, but my knowledge says this is not a safe file.

Please someone give to me a reply for change my mind!

Flags: https://prnt.sc/pOCZRQPutfCf

**Comment by Deltwin** [comment]
Are you a dumb or what? You see that this software is open-source right?

**Comment by JayBe13** [comment]
@DAVIDEGAMERS33 

Wait! You created an account just to post this?

**I am about to impart some knowledge.**

Do you actually think that 5 out of 70 antivirus-flagged DCSB means "This is 100% A Virus (Certified by Virustotal)"? That is just over 7.1%. Let me put that in perspective. Based on what you saying, the following quoted statement from the National Institute of Mental Health means 100% of the U.S. population age 18 and older has major depressive disorder: "Major depressive disorder affects approximately 17.3 million American adults, or about 7.1% of the U.S. population age 18 and older, in a given year." (National Institute of Mental Health, "Major Depression," 2017).

Please do some research before you post that something is 100% certified. What you are seeing is called a false positive. False positives happen when an antivirus program flags a secure piece of software as malware. While some antivirus programs raise fewer false positives than others, no antivirus program is 100% immune to false positives. Antivirus programs have a fairly broad set of criteria for deciding whether or not a file is safe.


**TLDR; False Positive!**

**Comment by Maxx1mus** [comment]
Of course is a Virus. this app dont work

**Comment by TecnikOfficial** [comment]
APP WORKS FINE AND ITS NOT A VIRUS ... i hav been using it for quite a while now
